packages/protocol/migrationsConfig.js

```diff
@@ -82,20 +82,20 @@ const DefaultConfig = {
     frozen: true,
   },
   governance: {
-    queueExpiry: 4 * WEEK,
-    dequeueFrequency: WEEK,
-    concurrentProposals: 10,
-    approvalStageDuration: 3 * DAY,
-    referendumStageDuration: WEEK,
-    executionStageDuration: WEEK,
+    queueExpiry: WEEK, // Change to 4 weeks once mainnet activated
+    dequeueFrequency: DAY, // Change to 1 week once mainnet activated
+    concurrentProposals: 3, // Change to 10 once mainnet activated
+    approvalStageDuration: DAY, // Change to 3 days once mainnet activated
+    referendumStageDuration: DAY, // Change to 1 week once mainnet activated
+    executionStageDuration: DAY, // Change to 1 week once mainnet activated
```

Comment:
Once the execution stage ends, the proposal will expire, even though it was already approved. I think it'd be better to keep this as a week because we (or anyone) could execute the proposal as soon as the it's ready, or if something happens that we don't expect, the proposal would stay valid for up to 7 days.
```suggestion
    executionStageDuration: WEEK,
```

Reply:
+1 -- I will update